liberty-maven-plugin/src/it/basic-runtime-properties-pom-override-it/src/test/java/net/wasdev/wlp/maven/test/app/LibertyRuntimePropertiesIT.java

```diff
@@ -46,7 +46,7 @@ public void buildLogCheck() throws Exception {
         final String ARTIFACTID_MESSAGE = "[INFO] The runtimeArtifact artifactId openliberty-webProfile8 is overwritten by the liberty.runtime.artifactId value openliberty-runtime.";
         boolean ARTIFACTID_MESSAGE_FOUND = false;
 
-        final String VERSION_MESSAGE = "[INFO] The runtimeArtifact version 19.0.0.6 is overwritten by the liberty.runtime.version value 19.0.0.7.";
+        final String VERSION_MESSAGE = "[INFO] The runtimeArtifact version 19.0.0.6 is overwritten by the liberty.runtime.version value 21.0.0.3.";
```

Comment:
With the change in BasicSupport to default to `21.0.0.9`, shouldn't this log message change to reference `21.0.0.9` instead of `19.0.0.6`?

Reply:
The version set in the test pom.xml is still `19.0.0.6`.